### PR TITLE
Parse Items from OpenSea URLs

### PR DESCRIPTION
**Pull request by 0xjimmy**
Still need to check network compared to opensea URL and prompt user if not matching wallet.

### CODE REVIEW

**Comment by KillariDev** [comment]
@epheph, this is still a draft?

**Comment by epheph** [comment]
> @epheph, this is still a draft?

oh sorry, maybe it is, i just want it bad :)

File: app/ts/components/TokenPicker.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Signal, useSignal } from "@preact/signals";
 import { getAddress } from "ethers"
 import { JSX } from "preact/jsx-runtime";
```
Comment (KillariDev) [inline app/ts/components/TokenPicker.tsx]:
```suggestion
import { getAddress } from 'ethers'
import { JSX } from 'preact/jsx-runtime'
```

File: app/ts/components/Transfer.tsx [hunk 2]
```diff
@@ -1,19 +1,22 @@
-import { Signal, useComputed, useSignal, useSignalEffect } from "@preact/signals";
+import { batch, Signal, useComputed, useSignal, useSignalEffect } from "@preact/signals";
```
Comment (KillariDev) [inline app/ts/components/Transfer.tsx]:
```suggestion
import { batch, Signal, useComputed, useSignal, useSignalEffect } from '@preact/signals'
```

File: app/ts/components/Transfer.tsx [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
 import { getAddress } from "ethers";
 import { JSX } from "preact/jsx-runtime";
 import { connectBrowserProvider, ProviderStore } from "../library/provider.js";
-import { detectNft } from "../library/identifyTokens.js"
+import { itentifyAddress } from "../library/identifyTokens.js"
```
Comment (KillariDev) [inline app/ts/components/Transfer.tsx]:
```suggestion
import { itentifyAddress } from '../library/identifyTokens.js'
```

File: app/ts/components/Transfer.tsx [hunk 4]
```diff
@@ -1,19 +1,22 @@
...
 import { BlockInfo } from "../library/types.js";
 import { Button } from "./Button.js";
 import { TokenPicker } from "./TokenPicker.js";
 import { transferNft } from "../library/transactions.js";
 import Blockie from "./Blockie.js";
+import { createRef } from "preact";
+import { knownNetworks } from "../library/networks.js";
```
Comment (KillariDev) [inline app/ts/components/Transfer.tsx]:
```suggestion
import { createRef } from 'preact'
import { knownNetworks } from '../library/networks.js'
```

File: app/ts/library/identifyTokens.ts [hunk 5]
```diff
@@ -1,47 +1,143 @@
 import { Contract, Provider, Interface, ZeroAddress, BytesLike } from "ethers";
-import { ERC721ABI, MulticallABI } from "./abi.js";
+import { ERC1155ABI, ERC20ABI, ERC721ABI, MulticallABI } from "./abi.js";
```
Comment (KillariDev) [inline app/ts/library/identifyTokens.ts]:
```suggestion
import { ERC1155ABI, ERC20ABI, ERC721ABI, MulticallABI } from './abi.js'
```

**Review by KillariDev — APPROVED** [review]

**Pull request merged**